src/test/java/simpaths/model/PersonTest.java

```diff
@@ -0,0 +1,110 @@
+package simpaths.model;
+
+import org.junit.jupiter.api.*;
+import simpaths.data.Parameters;
+import simpaths.model.enums.Country;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@DisplayName("Person class")
+public class PersonTest {
+
+    static Person testPerson;
+
+    @BeforeAll
+    static void setup() {
+        testPerson = new Person(true);
+        Parameters.loadParameters(Country.UK, 100, false, false, false, false, false, false, false, 2020, 2020, 2020, 1.,1., false, false);
+    }
+
+    @Nested
+    @DisplayName("EQ5D process")
+    class Eq5dTests {
+
+        @Nested
+        @DisplayName("With default parameters")
+        class WithDefaultParameters {
+
+            @Test
+            @DisplayName("Calculates low score correctly using Lawrence and Fleishman coefficients")
+            public void calculatesLowScoreCorrectly() {
+
+
+                testPerson.setDhe_mcs(1.);
+                testPerson.setDhe_pcs(1.);
+
+                testPerson.onEvent(Person.Processes.HealthEQ5D);
+
+                assertEquals(-0.594, testPerson.getHe_eq5d());
+
+            }
+
+            @Test
+            @DisplayName("Calculates high score correctly using Lawrence and Fleishman coefficients")
+            public void calculatesHighScoreCorrectly()  {
+
+
+                testPerson.setDhe_mcs(100.);
+                testPerson.setDhe_pcs(100.);
+
+                testPerson.onEvent(Person.Processes.HealthEQ5D);
+
+                assertEquals(1, testPerson.getHe_eq5d());
+
+            }
+
+        }
+
+
+        @Nested
+        @DisplayName("With eq5dConversionParameters set to 'franks'")
+        class WithFranksParameters {
+
+            @BeforeAll
+            public static void setupFranksCoefficients() {
+
+                Parameters.eq5dConversionParameters = "franks";
+                Parameters.loadEQ5DParameters("UK", 8);
+
+            }
+
+            @AfterAll
+            public static void restoreDefaultCoefficients() {
+
+                Parameters.eq5dConversionParameters = "lawrence";
+                Parameters.loadEQ5DParameters("UK", 8);
+
+            }
```

Comment:
@igelstorm I've extracted the lightweight 'update only EQ5D parameters' method to run as a before/after chunk here. That way parameters can persist as setup across all Person tests?